README.md

```diff
@@ -100,9 +102,19 @@ $ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operato
 If you want to make changes to the HCO, here's how you can test your changes
 through [OLM](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#installing-olm).
 
+The make target `cluster-sync-olm` automates the steps in this section:
+```shell
+$ make cluster-sync-olm
+```
+
+If you would rather want to understand the details, you can find it below.
```

Comment:
I think this is a bit confusing explaining that the steps below are not necessary
Can you try maybe using subsections?

Reply:
By subsections, do you mean something like:
* TL;DR (this has only the one line about using make target)
* Longer version (this should have rest of the details mentioned)

Reply:
yes, something like that sounds good